### ISSUE TITLE
[Feature]: Steering wheel controls Support.

### ISSUE THREAD

**Issue by vickysuman** [fast4x/RiMusic#1974]
### Description

 Steering wheel controls are not working with the app.
I installed the app on android head unit.
Love the app, UI, features, Really good job.
Please add this feature also.
Thanks

### Suggested Solution

Please add the support for Steering wheel controls.
Thanks

**Comment by fast4x** [comment]
you mean  steering wheel in Android Auto?

**Comment by vickysuman** [comment]
> you mean steering wheel in Android Auto?

no, I installed it in the head unit, that's running on android. And the steering controls works with other apps, like to play/pause or skipping the songs.
Thanks
